miden-tx/tests/integration/scripts/faucet.rs

```diff
@@ -39,14 +39,14 @@ fn prove_faucet_contract_mint_fungible_asset_succeeds() {
 
     // CONSTRUCT AND EXECUTE TX (Success)
     // --------------------------------------------------------------------------------------------
-    let data_store = MockDataStore::with_existing(Some(faucet_account.clone()), Some(vec![]));
+    let data_store = TransactionContextBuilder::new(faucet_account.clone()).build();
```

Comment:
nit: here and in a few other places in this file the variable should probably be named `tx_context`.

Reply:
done